REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.5
 Compat 0.19.0
 Reexport
+StatsBase 0.14.1
```

Comment:
As noted elsewhere, I think this should be 0.14, right? Else Tony is going to scream at us. :-)